docker/nginx/libs/skynet/account.lua

```diff
@@ -14,6 +14,9 @@ local anon_limits = {
     ["registry"] = 250
 }
 
+-- basic caching of user data
+local curr_limits
```

Comment:
The idea behind this one is that we're hitting accounts twice on each request just to get the user's speed limits. Those calls are cached by accounts but it's still two network calls, which is super slow.

I'm writing this comment mostly to ask if any reviewer know whether these `local` values in Lua are somehow persisted between requests. If that is going we can't do this caching in such a simple case but something tells me that it's not the case. I'll try to research that as well but anything nginx tends to spiral down into shamanism pretty quickly.

Reply:
This is already cached in `ngx.var.account_limits` - we're not hitting the /limits twice unless the requests come at the exactly same time because we don't have queuing here.